### DISCUSSION TITLE
Tutorial erect and pico mix

### DISCUSSION THREAD

**Discussion by Szymonkruk12** [FunkinCrew/Funkin#3541] [Ideas]
please add this because i want it

**Comment by JVNpixels** [comment]
Tutorial Erect was one of the first songs Kawai Sprite made for the erect remixes Dave said on a stream, so it will be coming, for the Pico Mix there is no current clues or evidence, personally I think maybe not since the player knows how to play the game by the time they unlock Pico as a playable character.

**Reply by isakube** [reply]
would be sick af to hear pico go off on tutorial type beat tho
